src/types.rs

```diff
@@ -200,6 +200,9 @@ pub struct NullAlias;
 #[derive(Debug, Clone)]
 pub enum Keyword {
     Null,
+    CurrentDate,
+    CurrentTime,
+    CurrentTimestamp,
     Custom(DynIden),
```

Comment:
Hey @ikrivosheev, thanks for the PR!! I think we can safely remove `Keyword::Custom`? Since no one is using it and it essentially did nothing.